sigma/backends/loki/loki.py

```diff
@@ -124,15 +124,29 @@ def finalize_expression(self) -> str:
 class LogQLDeferredOrUnboundExpression(DeferredQueryExpression):
     """'Defer' unbounded OR matching to pipelined command **BEFORE** main search expression."""
 
-    exprs: List[str]

```

Comment:
This is actually a case where Python types meant I was making assumptions that I shouldn't have! I'm a little surprised mypy didn't catch this in `convert_condition_or` - arg.value is always going to be a SigmaString (not a str) until I made this change - possibly the way I create the LogQLDeferredOrUnboundExpression prevented it from doing so?

Reply:
Mypy is not bullet-proof and there are many edge case it can't detect, especially in nested types.